tools/boards.txt.py

```diff
@@ -542,7 +542,7 @@
             'flashfreq_40',
             '4M',
             ],
-        'serial': '57',
+      # 'serial': '57',
```

Comment:
Why disabling the hardcoded serial speeds?  If it's 921 it will silently use 400k.  If it's 56K, there might be a HW reason for it on those specific boards....

Reply:
(like a really, really cheap USB<->serial chip, or high resistance/capacitance on the lines due to something else like LEDs sharing the pin, etc.)

Reply:
That's true. I was [wrongly](https://github.com/espressif/esptool#baud-rate) thinking esptool would try at lower rates.
I will revert this.